[Build 2025091413-4.3-fedora] openQA test fails in TC_10_RestoreInDispVM_fedora-43-xfce

## Observation

openQA test in scenario qubesos-4.3-pull-requests-x86_64-system_tests_backupdispvm@64bit fails in
[TC_10_RestoreInDispVM_fedora-43-xfce](https://openqa.qubes-os.org/tests/152963/modules/TC_10_RestoreInDispVM_fedora-43-xfce/steps/1)

```
# test_000_basic_backup
# failure: 

# timestamp 2025-09-14T16:11:12.349635
Traceback (most recent call last):
  File "/usr/lib/python3.13/site-packages/qubes/tests/integ/backupdispvm.py", line 102, in restore_backup
    backup_log = self.loop.run_until_complete(
        self.loop.run_in_executor(None, restore_in_dispvm.run)
    )
  File "/usr/lib64/python3.13/asyncio/base_events.py", line 725, in run_until_complete
    return future.result()
           ~~~~~~~~~~~~~^^
  File "/usr/lib64/python3.13/concurrent/futures/thread.py", line 59, in run
    result = self.fn(*self.args, **self.kwargs)
qubesadmin.exc.BackupRestoreError: qvm-backup-restore failed with 1

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib64/python3.13/contextlib.py", line 85, in inner
    return func(*args, **kwds)
  File "/usr/lib/python3.13/site-packages/qubes/tests/integ/backupdispvm.py", line 128, in test_000_basic_backup
    self.restore_backup(source=backup_path, appvm=self.backupvm.name)
    ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/site-packages/qubes/tests/integ/backupdispvm.py", line 106, in restore_backup
    self.fail(str(e) + " backup log: " + e.backup_log.decode())
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: qvm-backup-restore failed with 1 backup log: app: Checking backup content...
ERROR
Traceback (most recent call last):
  File "/usr/lib/python3.14/site-packages/qubesadmin/backup/restore.py", line 451, in run
    self.__run__()
    ............^^
  File "/usr/lib/python3.14/site-packages/qubesadmin/backup/restore.py", line 725, in __run__
    self.import_process.start()
    .........................^^
  File "/usr/lib64/python3.14/multiprocessing/process.py", line 121, in start
    self._popen = self._Popen(self)
                  ...........^^^^^^
  File "/usr/lib64/python3.14/multiprocessing/context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
           .............................................^^^^^^^^^^^^^
  File "/usr/lib64/python3.14/multiprocessing/context.py", line 300, in _Popen
    return Popen(process_obj)
  File "/usr/lib64/python3.14/multiprocessing/popen_forkserver.py", line 35, in __init__
    super().__init__(process_obj)
    ................^^^^^^^^^^^^^
  File "/usr/lib64/python3.14/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
    ............^^^^^^^^^^^^^
  File "/usr/lib64/python3.14/multiprocessing/popen_forkserver.py", line 47, in _launch
    reduction.dump(process_obj, buf)
    ..............^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.14/multiprocessing/reduction.py", line 60, in dump
    ForkingPickler(file, protocol).dump(obj)
    ...................................^^^^^
TypeError: cannot pickle '_thread.lock' object
when serializing dict item '_waitpid_lock'
when serializing subprocess.Popen state
when serializing subprocess.Popen object
when serializing dict item 'tar2_process'
when serializing qubesadmin.backup.restore.ExtractWorker3 state
when serializing qubesadmin.backup.restore.ExtractWorker3 object
when serializing tuple item 0
when serializing method reconstructor arguments
when serializing method object
when serializing dict item '_target'
when serializing multiprocessing.context.Process state
when serializing multiprocessing.context.Process object
Process ExtractWorker3-1:
Traceback (most recent call last):
  File "/usr/lib64/python3.14/multiprocessing/process.py", line 320, in _bootstrap
    self.run()
    ........^^
  File "/usr/lib/python3.14/site-packages/qubesadmin/backup/restore.py", line 451, in run
    self.__run__()
    ............^^
  File "/usr/lib/python3.14/site-packages/qubesadmin/backup/restore.py", line 725, in __run__
    self.import_process.start()
    .........................^^
  File "/usr/lib64/python3.14/multiprocessing/process.py", line 121, in start
    self._popen = self._Popen(self)
                  ...........^^^^^^
  File "/usr/lib64/python3.14/multiprocessing/context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
           .............................................^^^^^^^^^^^^^
  File "/usr/lib64/python3.14/multiprocessing/context.py", line 300, in _Popen
    return Popen(process_obj)
  File "/usr/lib64/python3.14/multiprocessing/popen_forkserver.py", line 35, in __init__
    super().__init__(process_obj)
    ................^^^^^^^^^^^^^
  File "/usr/lib64/python3.14/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
    ............^^^^^^^^^^^^^
  File "/usr/lib64/python3.14/multiprocessing/popen_forkserver.py", line 47, in _launch
    reduction.dump(process_obj, buf)
    ..............^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.14/multiprocessing/reduction.py", line 60, in dump
    ForkingPickler(file, protocol).dump(obj)
    ...................................^^^^^
TypeError: cannot pickle '_thread.lock' object
when serializing dict item '_waitpid_lock'
when serializing subprocess.Popen state
when serializing subprocess.Popen object
when serializing dict item 'tar2_process'
when serializing qubesadmin.backup.restore.ExtractWorker3 state
when serializing qubesadmin.backup.restore.ExtractWorker3 object
when serializing tuple item 0
when serializing method reconstructor arguments
when serializing method object
when serializing dict item '_target'
when serializing multiprocessing.context.Process state
when serializing multiprocessing.context.Process object
qvm-backup-restore: error: unable to extract the qubes backup. Check extracting process errors.
exit code: 1


# system-out: 

```

## Test suite description



## Reproducible

Fails since (at least) Build [2025091404-4.3-fedora](https://openqa.qubes-os.org/tests/152879)


## Expected result

Last good: [2025091203-4.3](https://openqa.qubes-os.org/tests/152832) (or more recent)


## Further details

Always latest result in this scenario: [latest](https://openqa.qubes-os.org/tests/latest?arch=x86_64&distri=qubesos&flavor=pull-requests&machine=64bit&test=system_tests_backupdispvm&version=4.3)

Related to https://github.com/python/cpython/issues/84559

--- a/qubesadmin/backup/restore.py
+++ b/qubesadmin/backup/restore.py
@@ -55,6 +55,10 @@ from qubesadmin.exc import QubesException
 from qubesadmin.utils import size_to_human
 
 
+# Python 3.14 in Fedora 43 changes the default start method away from fork
+if multiprocessing.get_start_method(allow_none=True) != "fork":
+    multiprocessing.set_start_method("fork")
+
 # must be picklable
 QUEUE_FINISHED = "!!!FINISHED"
 QUEUE_ERROR = "!!!ERROR"

